urls.py

```diff
@@ -61,6 +61,16 @@
     'distro/core/qiime20176-1497566713.zip':
         'https://s3-us-west-2.amazonaws.com/qiime2-data/distro/core/qiime20176-1497566713.zip',
 
+    # 2017.7 DISTRO
+    'distro/core/qiime2-2017.7-conda-osx-64.txt':
+        'https://raw.githubusercontent.com/qiime2/environment-files/master/2017.6/release/qiime2-2017.7-conda-osx-64.txt',
```

Comment:
I think these OS X and Linux install files should be using the 2017.7 staging environment files, then we can update the links in the upcoming "part 2" PR to point to release env files.

For example, https://raw.githubusercontent.com/qiime2/environment-files/master/2017.7/staging/qiime2-2017.7-conda-linux-64.txt